client/modules/core/helpers/permissions.js

```diff
@@ -59,6 +59,10 @@ Template.registerHelper("hasDashboardAccess", function () {
   return Reaction.hasDashboardAccess();
 });
 
+Template.registerHelper("hasDashboardAccessForAnyShop", function () {
+  return Reaction.hasPermissionsForAnyShop();
```

Comment:
I searched for this function `hasPermissionsForAnyShop` and couldn't find it. Is it already on the marketplace branch? Or there's some typo here? Is it meant to be
```
Template.registerHelper("hasDashboardAccessForAnyShop", function () {
  return Reaction.hasDashboardAccessForAnyShop();
});
```